.circleci/config.yml

```
@@ -0,0 +1,199 @@
+version: 2.1
+orbs:
+  aws-s3: circleci/aws-s3@1.0.9
+
+environment:
+  DEBUG: true
+
+# -------------------------------------------------------------------------
+# jobs definition ---------------------------------------------------------
+# -------------------------------------------------------------------------
+set_default_excuter: &set_default_excuter
+  docker:
+    - image: node:carbon-alpine
+  working_directory: ~/select-frontend
+
+restore_saved_cache: &restore_saved_cache
+  restore_cache:
+    key: select-frontend-{{ checksum "package.json" }}
```

키를 하나만 지정하면 새 디펜던시가 생긴 경우 맞는 캐시 키를 찾지 못해서 처음부터 설치하게 됩니다.

```suggestion
    keys:
      - select-frontend-{{ checksum "yarn.lock" }}
      - select-frontend-
```

---

음.. 그럼 `CIRCLE_BRANCH ` 로 하나더 지정해서 가져가는건 어떨까요??
브랜치 명이니 해당 브랜치가 끝나기 전까지는 공통으로 하나를 가져갈 수 있을 것 같은데요